lib/Util/Aggregation.php

```diff
@@ -136,7 +136,15 @@ public function getOptimalAggregationLevel($uuid, $targetLevel = null) {
 			   'WHERE uuid = ? ';
 		if ($targetLevel) {
 			$sqlParameters[] = self::getAggregationLevelTypeValue($targetLevel);
```

Comment:
Might also simplify this to avoid the double value flipping from numeric index to actual level. The function is only ever used in a single place apart from testing.

Reply:
resolved by 078b95f